### DISCUSSION TITLE
Error: Unknown environment names 'tasmota32-sensors'

### DISCUSSION THREAD

**Discussion by 001oc** [arendst/Tasmota#17839] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Using vscode with pio.
When choosing/activating 'tasmota32-sensors' from platformio_override.ini the following error will thrown when trying to compile:

`Error: Unknown environment names 'tasmota32-sensors'. Valid names are 'tasmota32s3cdc-ocd, tasmota-CN, tasmota32-PT, tasmota32c3-safeboot, tasmota-SK, tasmota32s3-qio_opi-all, tasmota32-core2, tasmota32-AF, tasmota-TR, tasmota32-FY, tasmota-TW, tasmota32-mi32-homebridge, 
tasmota-knx, tasmota32solo1-ocd, tasmota-rangeextender, tasmota32-nspanel, tasmota32c3-mi32-homebridge, tasmota-HU, tasmota-BG, tasmota32s3-safeboot, tasmota32-PL, tasmota32s2cdc-safeboot, tasmota-NL, tasmota32-bluetooth, tasmota32-CN, tasmota32-display, tasmota32s3-file, tasmota-RO, tasmota-SE, tasmota32-BR, tasmota32s3cdc, tasmota-DE, tasmota32s3-bluetooth, tasmota-zbbridge, tasmota32s2-safeboot, tasmota-ES, tasmota32-KO, tasmota-minimal, tasmota4M, tasmota-GR, tasmota32solo1, tasmota32-GR, tasmota32-IT, tasmota-KO, tasmota32-VN, tasmota-display, tasmota32-rangeextender, tasmota-FY, tasmota-lite, tasmota32c3cdc, tasmota32-ocd, tasmota-CZ, tasmota32-RU, tasmota-VN, tasmota32-ES, tasmota32c3-bluetooth, tasmota-ir, tasmota32, tasmota32s3-mi32-homebridge, tasmota32-HE, tasmota-FR, tasmota32solo1-safeboot, tasmota32-SK, tasmota-HE, tasmota32-FR, tasmota32c3cdc-ocd, tasmota-RU, tasmota32c3cdc-safeboot, tasmota32s3, tasmota-IT, tasmota32-webcam, tasmota32s3cdc-safeboot, tasmota-PT, tasmota32-AD, tasmota32s2-ocd, tasmota-AD, tasmota32-HU, tasmota32-CZ, tasmota32-NL, tasmota-AF, tasmota32-DE, tasmota32-safeboot, tasmota32-lvgl, tasmota32-debug, tasmota32-zbbrdgpro, tasmota32-RO, tasmota32-UK, tasmota32-SE, tasmota32-BG, tasmota, tasmota-sensors, tasmota32-odroidgo, tasmota-UK, tasmota-PL, tasmota32-ir, tasmota32c3, tasmota-debug, tasmota32-TW, tasmota32_base, tasmota-zigbee, tasmota32s2, tasmota32s2cdc, tasmota32-TR, tasmota-BR'
`
As one can see, as valid names there is 'tasmota_sensors' but no 'tasmota32_sensors' although it is chooseable from the list in platformio_override.ini.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): ESP32 Devkit but that is not relevant as the problem accours at compiletime.
- [x] Tasmota binary firmware version number used: 12.3.1
  - [] Pre-compiled
  - [x] Self-compiled
- [x] Flashing tools used: vscode pio esptool
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`: not applicable as it is a compiletime error
```lua
  Configuration output here: not applicable as it is a compiletimeerror


```
- [x] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`: not applicable as it is a compiletime error
```lua
  Rules output here: not applicable as it is a compiletime error


```
- [x] Provide the output of this command: `Status 0`: not applicable as it is a compiletime error
```lua
  STATUS 0 output here: not applicable as it is a compiletime error


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log: not applicable as it is a compiletime error
```lua
  Console output here: not applicable as it is a compiletime error


```

### TO REPRODUCE
_Steps to reproduce the behavior:_
choose 'tasmota32_sensors' from platformio_override.ini

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._
compile 'tasmota32_sensors'

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._
not applicable as it is a compiletime error

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
my setup is as follows:
esp32 devkit (not relevant as it is a compiletime error)
vscode
pio

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by barbudor** [comment, marked as answer]
There is no more tasmota32-sensors
Due to the capacity of the ESP32, 'sensors' is now part of tasmota32 base.

**Comment by Jason2866** [comment]
Removing the outdated leftover in the sample file, just now. Thx for the info.

**Reply by 001oc** [reply]
I would prefere/suggest to mark 'tasmota32_sensors' as outdated (idealy with a short comment that is implemented in the base version of tasmota32) instead of removing it completely as it is not implemented in ESP8266 base version. If one is searching sensors for tasmota32 but is finding it solely for tasmota, you will get probably the next question to implement sensors at tasmota32 too ;-)

**Comment by Jason2866** [comment]
No, in actual used platformio files there is and will be no comment of settings used in ancient times. By doing this it would be littered. Tasmota has an changelog file.
